### PR TITLE
Dockerfile for snippets repo

### PR DESCRIPTION
**Pull request by ypark234**
Add a dockerfile for CI tests on snippets repo.
I expect more dependencies to be added as needed.
Maybe it'd be better to be part of snippets repo itself?

### CODE REVIEW

File: snippets [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04
```
Comment (gonuke) [inline snippets:1]: Wow!!! Ubuntu 20.04!

File: snippets [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# Python version.
+ARG py_ver=3
+
+RUN apt update \
+    # Dependencies.
+    && apt -y install python${py_ver} python${py_ver}-distutils curl \
+    # Install pip.
+    && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
```
Comment (gonuke) [inline snippets]: Is this the preferred/recommended way to install `pip` vs `apt install`?

**Review by gonuke — COMMENTED** [review]
Just one little question - not sure if it will result in a change?

File: snippets (hunk 2 above)
Comment (ypark234) [reply snippets]: Definitely `apt install` is first-recommended way.
However, from my previous experiences, doing so wasn't always successful at installing pip, especially for python version 3. So I instead started following [this guide](https://pip.pypa.io/en/stable/installing/).
I haven't tried if `apt install` works in Ubuntu 20.04 without any error though..

Comment (ypark234) [reply snippets]: Just run a quick test:
- `sudo apt install python3-pip` works.
- `sudo apt install python-pip` fails with error: `E: Unable to locate package python-pip` - [see this](https://superuser.com/questions/1545380/ubuntu-20-04-e-unable-to-locate-package-python-pip)

Comment (ypark234) [reply snippets]: Good news is that even with Python2 being deprecated from Ubuntu 20.04, the above alternative way works okay, with little extra revision.

**Comment by gonuke** [comment]
If python2 is unavailable on U20.04, then why have python version as an argument in the Dockerfile?

**Comment by ypark234** [comment]
Yeah, I guess it's time to drop python2.

**Review by gonuke — APPROVED** [review]
Thanks @ypark234 - I'll also add this to autobuild on dockerhub

**Pull request merged**